[2024-07-15T15:00:57.369+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-15T15:00:57.399+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow_book_api_pipeline.get_book_data_from_API scheduled__2024-07-14T00:00:00+00:00 [queued]>
[2024-07-15T15:00:57.407+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow_book_api_pipeline.get_book_data_from_API scheduled__2024-07-14T00:00:00+00:00 [queued]>
[2024-07-15T15:00:57.408+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-15T15:00:57.430+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_book_data_from_API> on 2024-07-14 00:00:00+00:00
[2024-07-15T15:00:57.439+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=56) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-15T15:00:57.441+0000] {standard_task_runner.py:63} INFO - Started process 59 to run task
[2024-07-15T15:00:57.444+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', '***_book_api_pipeline', 'get_book_data_from_API', 'scheduled__2024-07-14T00:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/***_book_api_dags.py', '--cfg-path', '/tmp/tmpbx6vqcg1']
[2024-07-15T15:00:57.446+0000] {standard_task_runner.py:91} INFO - Job 55: Subtask get_book_data_from_API
[2024-07-15T15:00:57.509+0000] {task_command.py:426} INFO - Running <TaskInstance: airflow_book_api_pipeline.get_book_data_from_API scheduled__2024-07-14T00:00:00+00:00 [running]> on host e6b306b9c204
[2024-07-15T15:00:57.631+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='zainal' AIRFLOW_CTX_DAG_ID='***_book_api_pipeline' AIRFLOW_CTX_TASK_ID='get_book_data_from_API' AIRFLOW_CTX_EXECUTION_DATE='2024-07-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-14T00:00:00+00:00'
[2024-07-15T15:00:57.632+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-15T15:01:00.159+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-15T15:01:00.160+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-15T15:01:00.182+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=***_book_api_pipeline, task_id=get_book_data_from_API, run_id=scheduled__2024-07-14T00:00:00+00:00, execution_date=20240714T000000, start_date=20240715T150057, end_date=20240715T150100
[2024-07-15T15:01:00.234+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-15T15:01:00.267+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-15T15:01:00.277+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
